### ISSUE TITLE
[Bug Report][3.5.6] v-data-table-virtual doesn't virtualize rows when initialized with empty array of items

### ISSUE THREAD

**Issue by franciscohermida** [vuetifyjs/vuetify#19291]
### Environment
**Vuetify Version:** 3.5.6
**Vue Version:** 3.4.20
**Browsers:** Chrome 121.0.0.0
**OS:** Windows 10

### Steps to reproduce
- Initialize a v-data-table-virtual with empty array
- Add items to the array after it is initialized

### Expected Behavior
It should virtualize the rows after initialized with an empty array

### Actual Behavior
When initialized with an empty array it does not virtualize the rows when you add more items to the array

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqdVW1v2jAQ/iunrBNBghAC9IXRSu2Hbh82bRrVqoki1YALFokTOQ4Si/jvOydxSHEipH6B+O6ee+4en+1ZasVi2buPImeXUGtsTSQNIp9IevfCASa77opI0pVk4dPujgmZEF85AMYbSlZUxLcvVvH1YhUehjmUPfvX1g1l641E69B1tU0FdHfETyjaOQlo7rib9Op4saJJr1IeLuOlYJGEmMokAp/wNeaRSJkVz4IoFBJSEPStA8swiBJJV3CANxEG0MJ+W19UFkAfj2VWTQy3KtyezdvKpZyY/IkFNEykbbfh9g7SY/Gxk1WPoKLGh5DIwqiiDh3wXNfNcmmaQi3EzPJEKUgmfTqGlkLD0z6irQ4Qn6052mJJhMT1lu5xpURqYdpT5DSi2Jq95aGM2xU05asSG6uYOvB3ytdyA3bQhPSzgDroL8GWFOyLJmSk/HXAv5SIBsxeuQrIvKrcQolb0S3/A1CiaA1iSTGv9mQtj2EwKi15K2PwvNKU1YhBuFGuW1pVFRjmev3CVPZwQvtzSQn/QWqJPZO4UktBPGog1qYm4j/hnqzrSI+5NOnQIL2qJ+3fnCF9Jjv6O+G8hndo8pbpSl4v69ZsVm9HE++UkgdB6T88GqftXpsa6z0raYf1tH2NbaL9RsQiFF8TInA+T4hL8JF4dKQoiK8bdL4616/PlttHxs1ZHpizPDRmuUHlc7M8laEIUGeyrZur45EphT5KUDBffnCYnxlfTQnzTZXNIzQypvnmg9P8iMf2ia3MoRro/amobOytl23uuVl+d4VVnwm8yfSj9P5lEficCQ4zx3HuhSB7u49PiIOXY2y3505AIptVoqu3I6ZMAWHZTTlj8Dm/M528hzkcNESZHSUCIl4v0nJ5gE8XKTu86riiFOUvesK3LP+d9PLXF99a69CxBs7Q6d9Y6mPkXFrz/yHXXiQ=)

**Comment by RensBoeser** [comment]
This also happens on v-autocompletes when you fill the items array only after opening the dropdown (using update:menu).

A workaround for this is to initially have one item in the array at all times.

**Comment by romankaravia** [comment]
I believe this is a duplicate of #18806 (which also has more suggestions for workarounds).
